meerk40t/gui/wxmeerk40t.py

```diff
@@ -3348,6 +3349,8 @@ def collapse(self, node):
         if not item.IsOk():
             raise ValueError("Bad Item")
         self.wxtree.CollapseAllChildren(item)
+        if self.wxtree.GetItemParent(item) == self.wxtree.GetRootItem():
+            self.wxtree.Expand(item)
 
```

Comment:
Hey, they collapsed the root, they should live with it.

Reply:
That is why Open Source is so great. You get collaboration between heroic coder geniuses who don't give a hoot about non-geniuses screwing things up, and mere mortals who know that you need to put guard rails up for the less tech savvy users.